.github/shared-actions/windows-bazel-test/action.yml

```diff
@@ -27,17 +27,21 @@ runs:
       with:
         wsl_firewall_interface: 'vEthernet (WSL (Hyper-V firewall))'
 
+    - name: Installing pnpm (in WSL)
```

Comment:
Can this be done using `pnpm/action-setup`?

Reply:
No, because this is inside WSL